src/app/prt-staking/components/prt-section/prt-card.tsx

```diff
@@ -43,45 +56,55 @@ export function PrtCard({ onClick, token }: Props) {
       <div className='bg-ic-gray-50 border-ic-gray-300 my-3 w-full rounded-xl border px-3 py-5 text-sm'>
         <div className='text-ic-gray-950 mb-2 flex font-bold'>
           <div className='flex-1'>Your Staked PRTs</div>
-          <div>3.64 PRTs</div>
+          <div>{userStakedBalance ? `${userStakedBalance} PRTs` : ''}</div>
         </div>
         <div className='text-ic-gray-600 flex font-medium'>
           <div className='flex-1'>Your share of Pool</div>
-          <div className='font-bold'>0.031%</div>
+          <div className='font-bold'>
+            {userStakedBalance && poolStakedBalance
+              ? `${(Number(userStakedBalance / poolStakedBalance) * 100).toFixed(3)}%`
```

Comment:
Should be able to use `formatAmount` here?

Reply:
good call - updated 👍